src/main/java/com/es/phoneshop/model/product/PriceChange.java

```diff
@@ -0,0 +1,7 @@
+package com.es.phoneshop.model.product;
+
+import java.math.BigDecimal;
+import java.util.Date;
+
+public record PriceChange(Date dateOfChange, BigDecimal price) {
```

Comment:
why the price history works only for Samsung Galaxy S?

Reply:
it's because price changes for all products (except for Samsung Galaxy S) is set to empty lists (i will add price changes for other products)